docs/platform/howto/byoc/create-custom-cloud.md

```diff
@@ -812,6 +841,14 @@ Your new custom cloud is ready to use only after its status changes to
      --google-privilege-bearing-service-account-id "GENERATED_SERVICE_ACCOUNT_ID"
    ```
 
+   where
+
+   - `organization-id` is the identifier of your Aiven organization that is to be
+     connected with your own cloud account to create the custom cloud.
+   - `byoc-id` is the identifier of your custom cloud.
+   - `google-privilege-bearing-service-account-id` is the identifier of the service account
```

Comment:
Extracted from the output of the previous `terraform apply` or `terraform output` command.
Example: `projects/your-project/serviceAccounts/cce-cce0123456789a@your-project.iam.gserviceaccount.com`